### PR TITLE
Javascript javascript1 week3

### PR DESCRIPTION
**Pull request by sayali05**

### CODE REVIEW

File: javascript/javascript1/week2/main.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
+//Flight booking fullname function
+function getFullname(firstname, surname, gender, useFormalName) {
+	if (gender === "male" && useFormalName === true) {
+		return "Your client full name is Lord " + firstname + " " + surname;
```
Comment (naresh699) [inline javascript/javascript1/week2/main.js:4]: Please check 'template string ES6' either it can write variables in a better way or not.

Comment (naresh699) [reply javascript/javascript1/week2/main.js:4]: This code is fine, there is another approach below.

function not doing any work on 'first name, surname. I prefer to create variables that change on the basis of gender and return the entire common line with variables. so we can avoid multiple lines of code.

`let genderTitle =  '';
        genderTitle = gender === 'male' ? 'lord': 'queen';
        return 'Your client full name is ${genderTitle} ${firstname} ${surname}';

`

File: javascript/javascript1/week2/main.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+//Event application
+const weekdays = [
+	"Sunday",
+	"Monday",
+	"Tuesday",
+	"Wednesday",
+	"Thursday",
+	"Friday",
+	"Saturday",
```
Comment (naresh699) [inline javascript/javascript1/week2/main.js:28]: last index doesn't need ,

File: javascript/javascript1/week2/main.js [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+];
+function getEventWeekday(days) {
+	const todaysDate = new Date();
+	let today = todaysDate.getDay();
+	//console.log("Today is the " + weekdays[day]);
+
+	let eventDate = (today + days) % 7;
+	let eventweekday = weekdays[eventDate];
+	console.log(
+		`Today is ${weekdays[today]} and the event is in ${days} days. Therefore the event will be held on a ${eventweekday}.`
```
Comment (naresh699) [inline javascript/javascript1/week2/main.js:38]: this template string i was expecting in 'getFullname' function too

**Review by naresh699 — APPROVED** [review]
hi Sayali, Code is fine. My comments are just feedback for learning more. this is ready to merge.

- if writing fresh code, try to use ES6 features
- read about helper methods in Array or obj. Check whether this method changes the current array or makes a new array.
- read about variable hoisting,
- Format Code
- you used comments to describe the function, that's great.  when you have free time without hampering your learning please read about https://jsdoc.app/

File: javascript/javascript1/week2/main.js (hunk 1 above)
Comment (sayali05) [reply javascript/javascript1/week2/main.js:4]: i tried with ternary condition as per your review.

File: javascript/javascript1/week2/main.js (hunk 2 above)
Comment (sayali05) [reply javascript/javascript1/week2/main.js:28]: In my editor after saving file it automatically adding , in the last index

**Comment by sayali05** [comment]
Thank you so much for the feedback and your valuable time.

**Pull request merged**